<?xml version="1.0"?>
<launch>
    <node pkg="turtlesim" type="turtlesim_node" name="sim" />
    <node pkg="turtlesim" type="turtle_teleop_key" name="teleop" output="screen" />
    <param name="scale_linear" value="1" type="double" />
    <param name="scale_angular" value="1" type="double" />
    <node pkg="learning_tf2" type="turtle_tf2_broadcaster" args="/turtle1" name="turtle1_tf2_broadcaster" />
    <node pkg="learning_tf2" type="turtle_tf2_broadcaster" args="/turtle2" name="turtle2_tf2_broadcaster" />
    <node pkg="learning_tf2" type="turtle_tf2_listener" name="listener" />
</launch>
